doc/examples/applications/plot_human_mitosis.py

```diff
@@ -0,0 +1,176 @@
+# the cells. If it were, we could readily compute a mitotic index for this
+# sample:
+
+cells = image > thresholds[0]
+dividing = image > thresholds[1]
+labeled_cells = morphology.label(cells)
+labeled_dividing = morphology.label(dividing)
+naive_mi = labeled_dividing.max() / labeled_cells.max()
+print(naive_mi)
+
+#####################################################################
+# Whoa, this can't be! The number of dividing nuclei
+
+print(labeled_dividing.max())
+
+#####################################################################
+# is overestimated, while the total number of cells
+
+print(labeled_cells.max())
+
+#####################################################################
+# is underestimated.
+
+fig, ax = plt.subplots(ncols=3, figsize=(15, 5))
+ax[0].imshow(image)
+ax[0].set_title('Original')
+ax[0].axis('off')
+ax[2].imshow(cells)
+ax[2].set_title('Cells?')
+ax[2].axis('off')
+ax[1].imshow(dividing)
+ax[1].set_title('Dividing nuclei?')
+ax[1].axis('off')
+plt.show()
+
+#####################################################################
+# Count dividing nuclei
+# =====================
+# Clearly, not all connected regions in the middle plot are dividing nuclei.
+# On one hand, the second threshold (value of ``thresholds[1]``) appears to be
+# too low to separate those very bright areas corresponding to cell division
+# from relatively bright pixels otherwise present in many cells. On the other
+# hand, we want a smoother image, removing small spurious objects and,
+# possibly, merging clusters of neighbouring objects (some could correspond to
+# two emerging nuclei in one mother cell). In a way, the segmentation
+# challenge we are facing with dividing nuclei is the opposite of that with
+# (touching) cells.
+
+#####################################################################
+# To find suitable values for thresholds and filtering parameters, we proceed
+# by dichotomy, visually and manually.
+
+higher_threshold = 125
+dividing = image > higher_threshold
+
+smoother_dividing = filters.rank.mean(util.img_as_ubyte(dividing),
+                                      morphology.disk(4))
+
+binary_smoother_dividing = smoother_dividing > 20
+
+fig, ax = plt.subplots(figsize=(5, 5))
+ax.imshow(binary_smoother_dividing)
+ax.set_title('Cells in mitosis')
+ax.axis('off')
+plt.show()
+
+#####################################################################
+# We are left with
+cleaned_dividing = morphology.label(binary_smoother_dividing)
+print(cleaned_dividing.max())
+
+#####################################################################
+# dividing nuclei in this sample.
+
+#####################################################################
+# Segment cells
+# =============
+# To separate touching and overlapping cells, we resort to
+# :ref:`sphx_glr_auto_examples_segmentation_plot_watershed.py`.
+
+distance = ndi.distance_transform_edt(cells)
+
+local_maxi = feature.peak_local_max(distance, indices=False,
```

Comment:
`min_distance` is maybe easier to understand than `footprint` when the `footprint` is isotropic.

Reply:
Right. Thanks!